### PR TITLE
Added Babel to user get the final result working

### PR DESCRIPTION
**Pull request by DanZeuss**
Without Babel, the user cannot get the final result because we're using import * without transpiling it.

1. [Read and sign the CLA](https://cla.js.foundation/webpack/webpack.js.org). This needs to be done only once. PRs that haven't signed it won't be accepted.
2. Make sure your PR complies with [the writer's guide](https://webpack.js.org/writers-guide/).
3. Read through the PR diff carefully as sometimes this can reveal issues. The work will be reviewed, but this can save some effort.

### CODE REVIEW

File: content/get-started/index.md [hunk 1]
```diff
@@ -116,8 +116,15 @@ index.js  1.56 kB       0  [emitted]  main
 
 T> If you created a local `webpack@beta` build, be sure to reference `webpack` with `./node_modules/.bin/webpack` on the command line.
 
-Open `index.html` in your browser to see the result of a successful bundle.
-You should see a page with the following text: 'Hello webpack'.
+if you try to open `index.html` file in your browser, you will see a blank page, or a error in the console with the following message: `import declarations may only appear at top level of a module`. This means that webpack is bundling the non-transpiled ES6 code, which is why these import/export statements are being found. babel-loader must therefore not be transpiling what you expect.
```
Comment (bebraw) [inline content/get-started/index.md]: /s/if/If

**Comment by bebraw** [comment]
I wonder if it would be a better fix to drop *? Webpack should transpile imports for sure. It won't touch `const` etc. so perhaps having Babel around is good.

**Comment by DanZeuss** [comment]
@bebraw Thank you for giving me an opportunity to try.

**Comment by bebraw** [comment]
@SpaceK33z Do you want to have a quick look? Maybe introducing Babel at this point isn't a bad idea.

**Comment by bebraw** [comment]
@simon04 Any thoughts on this?

**Comment by simon04** [comment]
For inexperienced users this adds the need to dig into ES6, babel in addition to learning webpack. So I'd rather make the example ES5 compliant and maybe add a "Using webpack with ES6" just above the "Conclusion" section.

**Comment by bebraw** [comment]
@simon04 Great point. Pushing the Babel bits to a separate article like that would be ideal.

**Pull request closed without merging**